Functions.Templates/Resources/Resources.resx

```diff
@@ -1912,4 +1912,10 @@ function app.</value>
   <data name="SqlOuputBinding_description" xml:space="preserve">
     <value>A C# function that will be run on an HTTP trigger and takes a list of rows and upserts them into the user table (i.e. If a row doesn't already exist, it is added. If it does, it is updated).</value>
   </data>
+  <data name="SqlInputBinding_ts_description" xml:space="preserve">
+    <value>A TypeScript function that will be run on an HTTP trigger and returns the results of a provided SQL query.</value>

```

Comment:
There can be other triggers beyond HTTP

Reply:
But I suppose this template leverages an HTTP trigger, so oh well 😄

Reply:
Yeah, for now since we specifically create a template using an HTTP trigger this makes sense.

It would be nice if we could have our templates choose a base trigger as well, but I think that kind of functionality is out of scope unfortunately.